datafusion/functions-array/src/udf.rs

```diff
@@ -83,3 +84,121 @@ impl ScalarUDFImpl for ArrayToString {
         &self.aliases
     }
 }
+
+make_udf_function!(
+    Range,
+    range,
+    input diamilter,
```

Comment:
I think it should be
```suggestion
    start stop step,
```

And it will be expanded to
```rust
 pub fn range(start: Expr, stop: Expr, step: Expr) -> Expr {
        Expr::ScalarFunction(
            ScalarFunction::new_udf(
                range_udf(),
                <[_]>::into_vec(
                    #[rustc_box]
                    ::alloc::boxed::Box::new([start, stop, step]),
                ),
            ),
        )
    }
```
`cargo expand` can help to check it.

Reply:
got it